### PR TITLE
[HOPSWORKS-1382]-append add cron job to copy audit logs

### PR DESCRIPTION
**Pull request by ErmiasG**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -772,6 +772,19 @@
   end
 end
 
+if node['hopsworks']['audit_log_dump_enabled'].eql? "true"
```
Comment (kouzant) [inline recipes/default.rb]: It's better if you use `casecmp` which makes case-insensitive comparison

File: templates/default/dump_audit_logs_to_hdfs.sh.erb [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+LOGS_DIR=<%= @weblogs_dir %>
+HADOOP_HOME=<%= node['hops']['base_dir'] %>
+HDFS_WEBLOGS_DIR=<%= @remote_weblogs_dir %>
+
+CHECKPOINT=$LOGS_DIR/.checkpoint
+FILES_TO_DUMP=$LOGS_DIR/.files_to_dump
+REMOTE_FILE_PERMISSIONS=0750
+# This will match only files in the format of server_audit_log0.log
+REGEX=".*server_audit_log.*.log"
```
Comment (kouzant) [inline templates/default/dump_audit_logs_to_hdfs.sh.erb]: How about switching the general 'any' character with only numbers?

Comment (ErmiasG) [reply templates/default/dump_audit_logs_to_hdfs.sh.erb]: It can be with out any number too

File: recipes/default.rb (hunk 1 above)
Comment (ErmiasG) [reply recipes/default.rb]: do you mean 
if node['hopsworks']['audit_log_dump_enabled'].casecmp? "true"

Comment (ErmiasG) [reply recipes/default.rb]: or
node['hopsworks']['audit_log_dump_enabled'].casecmp("true") == 0

**Pull request merged**